config/sync/xmlsitemap.settings.yml

```diff
@@ -0,0 +1,16 @@
+_core:
+  default_config_hash: 58nI4vVTGv677fB42ySkQN9iTenfZvvW-xJlXODHdV4
+minimum_lifetime: 0
+xsl: 1
+prefetch_aliases: 1
+chunk_size: auto
+batch_limit: 100
+path: xmlsitemap
+frontpage_priority: 1.0
+frontpage_changefreq: 86400
+lastmod_format: 'Y-m-d\TH:i\Z'
```

Comment:
Their lastmod format was the 'Short' version in the Drupal advanced settings for XML Sitemap. You can see theirs here https://digital.gov/sitemap.xml

Reply:
updated date to be short version